tools/Cargo.lock

```diff
@@ -74,10 +74,11 @@ checksum = "64cb94155d965e3d37ffbbe7cc5b82c3dd79dd33bd48e536f73d2cfb8d85506f"
 
 [[package]]
 name = "assert-json-diff"
-version = "2.0.2"
+version = "1.1.0"
```

Comment:
This is a dev-dependency for `infrasys` and `aws-smithy-protocol-test`. From what I can tell the `infrasys` tests don't mine dropping to the same version as `aws-smithy`, so we can avoid pulling multiple versions.